wheelhouse.txt

```diff
@@ -1,2 +1,2 @@
-path.py>=8.0,<9.0
-jujubigdata>=5.0.0,<6.0.0
+jujubigdata>=6.0.0,<7.0.0
+jujuresources>=0.3.3,<1.0.0
```

Comment:
Is a wheelhouse needed for hive?  If charm build will merge hadoop-client's wheelhouse, i'd rather ensure jbd and jr come from client and drop the wheelhouse.txt from hive.